bootstrap/updater.go

```diff
@@ -1,3 +1,17 @@
+// Package bootstrap checks for the bootstrap/fallback data update from the
+// spacemesh administrator. This is intended as a short-term solution at the
+// beginning of the network deployment and should be removed once the network
+// is stable.
+//
+// The updater periodically checks for the latest update from a URL provided
+// by the spacemesh administrator, verifies the data, persists on disk and
+// notifies subscribers of a new update.
+//
+// Subscribers need to implement the following interface:
```

Comment:
What components would subscribe to this?

Reply:
beacon (for the fallback beacon value) and hare oracle (for the active set).